cmake/BuildInfo.cmake

```
@@ -0,0 +1,15 @@
+function(mrtrix_print_build_instructions)
+    set(message_list "\n")
+    file(RELATIVE_PATH relative_build_path "${PROJECT_SOURCE_DIR}" "${PROJECT_BINARY_DIR}")
+    list(APPEND message_list "MRtrix3 has been successfully configured for compilation.")
+    list(APPEND message_list "To build it, run 'cmake --build ${relative_build_path}'")
+
+    list(APPEND message_list "To reconfigure the project, run CMake again with the same arguments.")
+    list(APPEND message_list "If reconfiguration fails, remove the CMakeCache.txt file in the build "
+        "directory or run CMake adding the --fresh flag.")
```

```suggestion
        "directory, or include the --fresh flag when running cmake --build")
```

"Run CMake" is ambiguous as to whether running a configuration or a build.